### ISSUE TITLE
Error when imported data starts with equals sign ("=")

### ISSUE THREAD

**Issue by An-dir** [dfinke/ImportExcel#1459]
I have noticed that there are problems importing data if the imported text starts with a "=". There may be a better solution than the current one.

```
#How I did my tests with unsatisfactory results
# Testresult: Errors
"Col1;Col2`r`ntext;=|0" | ConvertFrom-Csv -Delimiter ";" | Export-Excel test1.xlsx -AutoFilter -AutoSize
# Testresult: Displays the '-character
"Col1;Col2`r`ntext;'=|0" | ConvertFrom-Csv -Delimiter ";" | Export-Excel -AutoFilter -AutoSize
# works but with ugly interpretation as text in a formula
"Col1;Col2`r`ntext;=`"=|0`"" | ConvertFrom-Csv -Delimiter ";" | Export-Excel -AutoFilter -AutoSize
```

In an old issue you wrote ( https://github.com/dfinke/ImportExcel/issues/1007#issuecomment-798961673 ) that Excel interprets this as a formula, but the following test looks promising that there may be a better solution:
```
# Here is how I expect it to be handled (delimiter is language dependend in your test):
"Col1;Col2`r`ntext;=|0" | ConvertFrom-Csv -Delimiter ";" | Export-Csv test.csv -Delimiter ";" -NoTypeInformation
start excel test.csv
```
Then save as xlsx has no problem with displaying it as string and reopen the xlsx without errors. The recognition as a formula starts when you try to make changes - then you can decide what you want to do with the data.

There can be more characters than the "=" character or "'" at the beginning, wich can cause problems when importing. 

It looks like in the "sharedStrings.xml" is missing "<si><t>=|0</t></si>" and maybe some other errors, like counter in this and maybe other files.

Maybe you can fix it so that when you import, it will not crash, but handle it like the xlsx via csv did

**Comment by dfinke** [comment]
please could you post a repro of the issue. data, script etc

**Comment by saviourofdp** [comment]
A simple example that I have come across is using the output of _Compare-Object_ 

```ps

**Comment by dfinke** [comment]
Yes. You need to handle that in your PowerShell script. Translate `==` to `equal` for example or prefix it with a`'` which is how native Excel enables you to start data with an `=` and it does not try to execute it as a function.

**Comment by saviourofdp** [comment]
Unless I'm doing something wrong, the `'` solution is not the same. If I create a workbook manually with a field "Test" with the value "'=" and create a second workbook with the script
```ps
[PSCustomObject]@{Test="'="} | Export-Excel -Path "test2.xlsx"
```
and then import them both, I get
```ps
Get-ChildItem . | select -ExpandProperty "FullName" | Import-Excel
```
<pre>
Test
----
=   
'=
</pre>

**Comment by dfinke** [comment]
Sorry, not sure what you are attempting. I was focused on the `==` if you try to enter that in the Excel UI it gives a message. I believe if you use it via PS Excel will see it as corrupt. So the `'=` is to prevent Excel from interpreting it as a formula

**Comment by saviourofdp** [comment]
Sorry what I was trying to get across was that if you type `=` into Excel and let Excel correct it to `'=` when you read that using this PowerShell module you get `=` as the value (ie no `'`). However if you set the value as `'=` using the PowerShell and then read it back using PowerShell you get `'=` (ie with a `'`)
